packages/vaadin-text-field/src/vaadin-text-field.d.ts

```diff
@@ -4,7 +4,7 @@ import { ControlStateMixin } from '@vaadin/vaadin-control-state-mixin/vaadin-con
 
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
-import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.js';
+import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
```

Comment:
**nit:** Sort the mixins in alphabetical order.

Reply:
Done.